python_ta/reporters/node_printers.py

```diff
@@ -78,7 +78,7 @@ def render_trailing_newlines(msg, _node, source_lines=None):
     start_line = msg.line - 1
     yield from render_context(start_line - 2, start_line, source_lines)
     yield from (
-        (line, slice(None, None), LineType.OTHER, source_lines[line - 1])
+        (line, slice(None, None), LineType.OTHER, source_lines[line - 1] + "# DELETE THIS LINE")
```

Comment:
Switch the line type to `ERROR`, which will trigger the highlighting to indicate this is an error.

But secondly, it actually seems that the current renderer doesn't actually show all of the extra blank lines (look at the example file to see the extra blank lines). So, please also update the logic of this renderer to highlight _all_ extra blank lines. You'll need to do some processing of `source_lines` to do so.